zircon.core/common/src/main/kotlin/org/hexworks/zircon/api/Components.kt

```diff
@@ -73,4 +63,10 @@ object Components {
 
     @JvmStatic
     fun vbox() = VBoxBuilder.newBuilder()
+
+    @JvmStatic
+    fun horizontalSlider() = HorizontalSliderBuilder.newBuilder()
```

Comment:
Why not have a `width`/`height` for the sliders as well?

Reply:
you're right I could have done that, but the size of each slider is forced to be height = steps + 1 or width = steps + 1 respectively. Do you think I should do it the other way around and restrict the number of steps from the size ?

Reply:
Oh I get it. But in this case I'm not sure I understand why changing the `steps` is possible? Zircon doesn't support `Component` resizing at the moment.

Reply:
there might be a semantics problem here, steps in the builder maps to numberOfSteps in the component proper. In the slider, a step represents a visual step of the slider, where the value is the numeric value of the slider.
So for example a (numberOfSteps = 10, range = 150) slider would be 10 tiles large visually, each step being range/numberOfSteps = 15. So if the currentValue is comprised between 0 and 15 the currentStep is 0, currentValue between 16-30 currentStep 1 etc.
The currentStep is used to render the slider cursor in the gutter.
Hope it makes sense

Reply:
I see, thanks!